### PR TITLE
Add pagination to Photos page

### PR DESCRIPTION
**Pull request by frak**
Please add [?w=1 to the diff URL](https://github.com/interactionresearchstudio/NaturewatchCameraServer/pull/11/files?w=1) of this PR to ignore the whitespace changes that were made by PyCharm, I had gotten too far into the changes before I realised I had cleaned it out of habit.

The requests to get the data are still slow when the file list is large, but caching a file directory that could be being written to would be difficult to keep in sync, so I have left this as is.

I have had to update the css file by hand as I don't know how to build your assets automatically.

### CODE REVIEW

**Comment by mikevanis** [comment]
Hi @frak, thanks for the PR. I have added information in the readme to walk you through how to compile the Less stylesheet. If you could move over the new CSS to the styles.less file and compile it into the styles.css file, that would be great.

**Comment by mikevanis** [comment]
Reviewing the code changes now, it looks like you managed to compile the less file already. Is that right?

**Comment by frak** [comment]
No, I just added the change manually to both places :) Not sure I need to build anything right now, but going forward I will use your instructions to rebuild the assets rather than working by hand.

**Comment by mikevanis** [comment]
Ok cool, I'll test this on my unit and merge away!

File: www/js/photos.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+            $('.move-backwards').removeClass('disabled');
+            $('.move-forwards').addClass('disabled');
+        } else {
+            console.log('Enabling next and prev')
+            $('.move-backwards').removeClass('disabled');
+            $('.move-forwards').removeClass('disabled');
+        }
+    }
+
+    function buildPaginationLinks(response) {
```
Comment (mikevanis) [inline www/js/photos.js]: I'm not sure why but if there aren't many photos (or no photos) in the gallery, negative pagination links are created. Screenshot attached. 
pagination

File: www/photos.html [hunk 2]
```diff
@@ -4,125 +4,138 @@
...
+                        <a class="page-link" href="javascript:void(0);" data-toggle="tooltip" data-placement="top"
+                           title="Forward to end">&gt;|</a>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <div class="col-md-3">
+            <nav id="page-size-nav" aria-label="Page size selector" class="float-right">
+                <ul class="pagination pagination-sm">
+                    <li class="page-item disabled"><span class="page-link">Size:</span></li>
```
Comment (mikevanis) [inline www/photos.html]: Instead of size, maybe a better wording would be "Photos per page: "

Comment (frak) [reply www/photos.html]: When I tried putting large words in here it broke the default styling and I wasn't really keen on trying to fix that :)

I'll look into another way of getting a label attached to this block and get back to you

File: www/js/photos.js (hunk 1 above)
Comment (frak) [reply www/js/photos.js]: Whoops! The thing I didn't want to test as I have ~3000 images on my Pi that need transferring.

I'll have a fix soon enough.

**Comment by frak** [comment]
Should all be good to go now

**Pull request merged**